addons/web_tour/static/src/tour_service/tour_compilers.js

```diff
@@ -294,8 +294,8 @@ export function compileStepAuto(stepIndex, step, options) {
                 skipAction = false;
                 console.log(`Tour ${tour.name} on step: '${describeStep(step)}'`);
                 if (!keepWatchBrowser) {
-                    clearTimeout(tourTimeout);
-                    tourTimeout = setTimeout(() => {
+                    browser.clearTimeout(tourTimeout);
+                    tourTimeout = browser.setTimeout(() => {
```

Comment:
Was this needed for this test or is this just an incidental cleanup?

Reply:
Yep, necessary for the last verifySteps